.github/workflows/assign.yaml

```diff
@@ -36,6 +39,6 @@ jobs:
           ref: master    
       - name: Installing the latest version of Go.
         uses: actions/setup-go@v2
-      # Run "assign-reviewers" subcommand on bot.
+      # Run "assign" subcommand on bot.
       - name: Assigning reviewers 
-        run: cd .github/workflows/ci && go run cmd/main.go --token=${{ secrets.GITHUB_TOKEN }} assign-reviewers
+        run: cd .github/workflows/robot && go run main.go -workflow=assign -token=${{ secrets.GITHUB_TOKEN }} -reviewers="${{ secrets.REVIEWERS }}
```

Comment:
I've been thinking about this, and perhaps we're thinking about the bot in the wrong way. We've been thinking about **_The_** bot, singular, when in reality there are multiple, semi-independent tasks that need to happen that could be structured as independent scripts. They'd all share some back-end code, yes, but splitting them out could let us simplify each script down to its essentials without having to worry as much about compatibility with the other tasks.

I've not sat down and thought on this deeply yet, so it may be a terrible idea!